pipeline_dp/pipeline_operations.py

```diff
@@ -153,23 +154,23 @@ def map(self, col, fn, stage_name: str = None):
     def map_tuple(self, col, fn, stage_name: str = None):
         return (fn(k, v) for k, v in col)
 
-    def map_values(self, col, fn, stage_name: str):
-        pass
+    def map_values(self, col, fn, stage_name: str = None):
```

Comment:
can you change this to `stage_name: Optional[str]`

Reply:
Has it been decided that we're adding mypy to the project? Why add the annotation for only one argument then?

Reply:
Done.